lib/JSON/Typist/DataPrinter.pm

```diff
@@ -0,0 +1,63 @@
+use strict;
+use warnings;
+package JSON::Typist::DataPrinter;
+
+# ABSTRACT: a helper for Data::Printer-ing JSON::Typist data
+
+use Data::Printer use_prototypes => 0;
+use Term::ANSIColor qw(colored);
+
+our $STRING_COLOR = 'ansi46';
+our $NUMBER_COLOR = 'bright_magenta';
+our $BOOL_COLOR   = 'ansi214';
+
+use Sub::Exporter -setup => [ qw( jdump ) ];
+
+=head1 SYNOPSIS
+
+  use JSON::Typist::DataPrinter qw( jdump );
+
+  my $data = get_typed_data_from_your_code();
+
+  say "I got data and here it is!";
+
+  say jdump($data); # ...and you get beautifully printed data
```

Comment:
Might be nice to include the beautifully printed data as an example in the pod

Reply:
Right now it's color-based, which means it's not actually really easy to show in Pod.  Probably this needs revising a few times before we're happy with it.  I'll put something in when it's more stable.